app/services/api/v3/ecf/participants_query.rb

```diff
@@ -1,12 +1,12 @@
 # frozen_string_literal: true
 
 module Api
-  module V1
-    module ECFParticipants
-      class Index
-        def initialize(lead_provider, params)
-          self.lead_provider = lead_provider
-          self.params        = params
+  module V3
```

Comment:
since we are using this in V1 and will be up to V3 could this stay in the V1 directory with the new structure you proposed?

Reply:
Hi @cwrw tks for your review. Addressed in https://github.com/DFE-Digital/early-careers-framework/pull/2687/commits/8faf4f1e53a01d9b0f5561e81bae9fd2b38cc091